src/openparse/tables/parse.py

```
@@ -95,30 +99,38 @@ def _ingest_with_table_transformers(
     pdf_as_imgs = doc_to_imgs(pdoc)
 
     pages_with_tables = {}
     for page_num, img in enumerate(pdf_as_imgs):
-        pages_with_tables[page_num] = find_table_bboxes(img, args.min_table_confidence)
+        pages_with_tables[page_num] = table_detector.detect(img, args.min_table_confidence)
 
     tables = []
     for page_num, table_bboxes in pages_with_tables.items():
-        page = pdoc[page_num]
-        page_dims = (page.rect.width, page.rect.height)
+        page: fitz.Page = pdoc[page_num]
```

Ruff (0.11.9)

Undefined name `fitz`

(F821)